lib/matplotlib/backends/backend_pdf.py

```diff
@@ -2423,8 +2425,11 @@ def __init__(self, filename, keep_empty=True):
         keep_empty: bool, optional
             If set to False, then empty pdf files will be deleted automatically
             when closed.
+        metadata: dictionary, optional
+            Information dictionary object (see PDF reference section 10.2.1
```

Comment:
I don't find this documentation particularly useful. You could you add at least some examples of metadata ?
Also, numpydoc format requires a space before the column : `metadata : dictionary, …`